src/main/java/util/Formatter.java

```diff
@@ -0,0 +1,30 @@
+package util;
+
+import java.util.List;
+
+public interface Formatter {
```

Comment:
Would it be better to implement the class as a concrete class instead? The intention of an interface is to act as a contract.